src/Storage/Listener/LocalListener.php

```diff
@@ -29,7 +27,7 @@ class LocalListener extends AbstractListener {
  * @var array
  */
 	protected $_defaultConfig = [
-		'pathBuilder' => 'BasePath',
+		'pathBuilder' => 'Local',
```

Comment:
I'm not sure any more but I think initially BasePathBuilder was already abstract, then I realized that very likely most adapters will be able to use it and just need to configure it a little different and made it a normal class. For example FTP won't behave much different than local. If BasePathBuilder can fulfil the needs of the listeners there is no need for another class?